src/Components/RecsListTable/index.js

```diff
@@ -52,9 +55,15 @@ const RecsListTable = () => {
   const page = filters.offset / filters.limit + 1;
   const [filteredRows, setFilteredRows] = useState([]);
   const [displayedRows, setDisplayedRows] = useState([]);
+  const setRecListSortIndex = (filters) =>
```

Comment:
I would rather not wrap the dispatch method with another function at all. Just call dispatch where the update of these parameters is needed.